packages/dso-toolkit/src/components/legend-item/legend-item.models.ts

```diff
@@ -0,0 +1,13 @@
+import { Selectable } from "../selectable/selectable.models";
+
+export interface LegendItem<TemplateFnReturnType> {
+  body: TemplateFnReturnType | undefined;
+  disabled: boolean;
+  dsoMouseEnter: () => void;
+  dsoMouseLeave: () => void;
```

Comment:
Als ik de discussie goed begrijp, zijn deze events ontstaan zodat een afnemer een tooltip kan tonen.

Wat zou je ervan vinden om dit onderdeel van Legend Item te maken?

```ts
disabledMessage?: string;
```

oid.

Reply:
Nee, dit is om de afnemer in staat te stellen "iets" te doen: in het geval van de viewer een highlight te tonen.

Reply:
Ooohhja wat goed.